tests/filestore-v2.8-stream-depth/suricata.yaml

```diff
@@ -0,0 +1,17 @@
+%YAML 1.1
+---
+
+outputs:
+  - eve-log:
+      enabled: yes
+      types:
+        - files
+  - file-store:
+      version: 2
+      enabled: yes
+      force-filestore: yes
+      stream-depth: 10000
```

Comment:
I get a failure with
`Sub test #1: FAIL : grep "SC_WARN_FILESTORE_CONFIG(330)] - file-store.stream-depth value 10000 has no effect since it's less than stream.reassembly.depth value" stdout | wc -l | xargs`